packages/help-center/src/components/help-center-search-results.tsx

```diff
@@ -374,12 +377,21 @@ function HelpSearchResults( {
 			{ ! searchQuery && <HelpCenterRecentConversations /> }
 			{ isSearching && ! searchResults.length && <PlaceholderLines lines={ placeholderLines } /> }
 			{ searchQuery && ! ( hasAPIResults || isSearching ) ? (
```

Comment:
No need to change.
I find that the condition is more readable if we avoid double negatives and clearly show that all three conditions must be true:
```
searchQuery && !hasAPIResults && !isSearching 
```